gnosis/safe-react
# style: Beamer drawer

## What it solves
Resolves #3472

## How this PR fixes it
Changes to the Beamer product id that has the custom css styles.

Gist with the custom css: https://gist.github.com/usame-algan/ed48b872686ae2bba12b7ba1848d6868

## How to test it

1. Open the safe
2. Accept the cookie banner
3. Click on "Whats new"
4. Observe the drawer with theme font, color and removed box-shadow

## Screenshots
<img width="412" alt="Screenshot 2022-02-16 at 10 23 16" src="https://user-images.githubusercontent.com/5880855/154235701-237eda02-e980-435e-8f11-dd214d2f6f91.png">
